### ISSUE TITLE
Trial man page is extremely out of date

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1887]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#1887|
|Type|defect|
|Created|2006-07-01 13:33:10Z|

It hasn't been updated since before slyphon rewrote Trial

<details><summary>Searchable metadata</summary>

```
trac-id__1887 1887
type__defect defect
reporter__jml jml
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__documentation documentation
time__1151760790000000 1151760790000000
changetime__1152170480000000 1152170480000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml removed owner<br>@jml set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)|@jerub commented|
|-|-|

Some things that need to be straightened out:

 * The SYNOPSIS is very rough, we probably want [OPTIONS] or something there instead of a random smattering of available options.
 * Make it clear that you have to specify what to test, and that it's not optional. The usage line in trial --help looks like ```trial [options] [[file|package|module|TestCase|testmethod]...]``` and is probably a good model.
 * The options should really be in alphabetical order.
 * The warning on --temp-directory isn't clear enough. it should really be *danger* because there's a tempation to do trial --temp-directory=/tmp instead of /tmp/mytrial/

There are options such as --supresswarnings that are omitted. Is this intentional (do I have an old version of trial?)

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml set owner to @jerub|
|-|-|

Changes implemented.  

--suppresswarnings is omitted because a) it's a noop and b) there is a branch pending review which removes the option entirely.

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)|@jerub commented|
|-|-|

Excellent. I have not tested that when you install or package twisted that this file will be installed to the correct place. Please test whatever process is in place for dealing with installing man pages, and assert that 'man trial' works as expected (I have been reading this docu using ```nroff -man trial.1|less```)

Things that I suggest you consider adding (these are optional, and not a blocker to merging) are a url to the documentation on how to write trial tests, and an 'EXAMPLE USAGE' section.

If you do make further alterations, pop it back to me for review, otherwise please merge.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml commented|
|-|-|

The processes for installing man pages are two-fold:

 1. Lore turns them into TeX and HTML
 1. Our Debian and RPM packages install them

Both aspects appear to work.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)|@jml set status to `closed`|
|-|-|

(In [17493]) Update trial manpage to reflect changes since Jan 2003

 * Author: jml
 * Reviewer: Jerub
 * Fixes [#1887](https://github.com/twisted/twisted/issues/1887)

Trial has been changed a _lot_ since 2003.  This documents the changes
in usage, adds some warnings and updates the copyright message.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
